backend.native/tests/build.gradle

```diff
@@ -164,6 +164,19 @@ task resultsTask() {
     }
 }
 
+boolean isExcluded(String task) {
+    // List of tests that fail due to unresolved comiler bugs
+    def excluded = [ "codegen/box/functions/bigArity", "codegen/box/functions/functionExpression" ]
```

Comment:
Can we exclude only individual test which shows FE bug in functionExpression?

Reply:
Only by adding the "IGNORE" keyword in the test. Otherwise it will turn on to maintaining of two different exclude strategies